manifests/init.pp

```diff
@@ -1,3 +1,29 @@
+# Install and manage the NATS messaging system
+#
+# @param routes_password
+# @param servers
+# @param client_port
+# @param monitor_port
+# @param cluster_port
+# @param max_payload_size
+# @param max_pending_size
+# @param max_connections
+# @param debug
+# @param trace
+# @param announce_cluster
+# @param binpath
+# @param configdir
+# @param piddir
+# @param binary_source
+# @param service_name
+# @param service_ensure
+# @param service_type
+# @param cert_file
+# @param key_file
+# @param ca_file
+# @param manage_collectd
+# @param limit_nofile
+#
```

Comment:
YARD generally does not like blank comments at the end, not sure if its the same with puppet but I'd prefer this was not here.  

Generally I'd question the value of these comments without details or types etc, puppet strings will automatically pull out the variables and types and everything for you, the thing it does on its own is more detailed than these comments so I doubt its needed.

Here's a example of it without the comment section https://devco.net/rip/puppet-nats/puppet_classes/nats.html